standards/collections.md

```diff
@@ -112,6 +116,34 @@ GET https://api.spscommerce.com/v1/example?limit=25&offset=0
 }
 ```
 
+- Invalid requests due to pagination validation **MUST** return a `400 Bad Request` status code following the standard [error schema](errors.md).
+
+```
+// CORRECT
+GET https://api.spscommerce.com/v1/books
+    ?limit=-2                               // limit cannot be a negative value
+// RESPONSE
+HTTP/1.1 400
+Content-Type: application/problem+json
+{
+    "title": "Invalid Data",
+    "status": 400,
+    "detail": "Missing content or invalid input provided.",
+    "instance": "/v1/books",
+    "requestId": "b6d9a290-9f20-465b-bcd3-4a5166eeb3d7",
+    "context": [
+       {
+            "code": "INPUT_MIN_VALUE",
+            "message": "Attribute 'limit' must be greater than or equal to 1.",
+            "field": "limit",
+            "source": "query",
+            "value": "0"
```

Comment:
Wouldn't this value be `-2` for this example? Or maybe I don't remember what the `value` field is used for :)

Reply:
haha, yep totally.